dashboard/templates/cards/statistics.html

```diff
@@ -1,40 +1,46 @@
 {% load duration i18n %}
 
-<div class="card card-dashboard card-statistics">
-    <div class="card-header">
-         <i class="icon icon-graph pull-left" aria-hidden="true"></i>
-        {% trans "Statistics" %}
-    </div>
-    <div class="card-body text-center">
-        <div id="statistics-carousel" class="carousel slide" data-interval="false">
-            <div class="carousel-inner">
-                {% for stat in stats %}
-                    <div class="carousel-item{% if forloop.counter == 1 %} active{% endif %}">
-                        <span class="card-title">
-                            {% if stat.stat %}
-                                {% if stat.type == 'duration' %}
-                                    {{ stat.stat|duration_string:'m' }}
-                                {% elif stat.type == 'float' %}
-                                    {{ stat.stat|floatformat }}
+{% if stats|length > 0 and not user.settings.dashboard_hide_empty %}
```

Comment:
I don't think `stats|length` would ever be empty here? You'd still need to determine whether or not to hide this card in the same manner as the others. The basis would just be somewhat different in that you'd probably only want to hide it if none of the stats functions produce results. So slightly more complicated.